templates/nginx-cfg.j2

```diff
@@ -22,4 +22,17 @@ server {
     proxy_redirect http:// https://;
     proxy_pass              http://jenkins;
   }
+
+  # Restrict the following paths to localhost (SSH tunneled users only)
+  location ~ /(api|restart|quietDown|user/jenkins|jnlpJars|script|asynchPeople|credential-store|systemInfo|view/All/newJob|) {
```

Comment:
I suspect that the last `|` at the end of the matching group could cause the regex to accidentally match the root `/` and deny access to the home page.